client/components/Header.jsx

```diff
@@ -38,7 +38,10 @@ const useStyles = makeStyles(theme => ({
     flexGrow: 1,
   },
   menuPaper: {
-    backgroundColor: colors.textPrimaryDark,
+    backgroundColor: colors.textPrimaryLight,
+    '& a': {
+      textDecoration: 'none',
+    },
   },
   menuItem: {
     color: 'white',
```

Comment:
Hi Shannan, looks good. 
However, could add the line `...theme.typeography.body2` to `menuItem` like so:

menuItem: {
    **...theme.typography.body2,**
    color: 'white',
    '&:hover': {
      textDecoration: 'underline',
    },
  },

This will change the font size of the menu items from `16px` to `14px`, which is the same size as the parent menu item values.

To find our standard theme sizes I usually just go here to [@theme/typography](https://github.com/hackforla/311-data/blob/8e875eb6c9a23ae291dcc98fe90b8b13d881a7f8/client/theme/typography.js#L43) and pick the one that is closest to the size I am looking for. In this case `body2` is `14px`. Thanks

Reply:
Oh good catch. Adding now.